recipe/meta.yaml

```diff
@@ -171,6 +108,10 @@ requirements:
     - _openmp_mutex                   # [linux]
```

Comment:
Can we use:
```
host:
- numpy {{ numpy }}
run:
- {{ pin_compatible('numpy') }} 
```

Reply:
Sure. [Here's](https://github.com/pytorch/pytorch/blob/e9ebda29d87ce0916ab08c06ab26fd3766a870e5/.ci/docker/common/install_conda.sh#L53) where the previous requirements came from. But they might be somewhat arbitrary/replaceable by our numpys